cigaradvisor/blocks/search-results/search-results.js

```diff
@@ -59,14 +59,10 @@ async function handleSearch(searchValue, wrapper, limit) {
   const filteredData = filterData(searchTerms, data);
 
   const articles = [];
-  const promises = [];
+  const allArticles = await loadPosts();
   filteredData.forEach((post) => {
-    promises.push(fetchPostsInfo(post.path));
```

Comment:
I'm not sure that this should be removed, i am wondering if all of the different locations in which this article list is found have been tested thoroughly.

Reply:
yes